keyboards/mysticworks/wyvern/info.json

```diff
@@ -0,0 +1,851 @@
+                {"x":13.25, "y":0},
+                {"x":14.25, "y":0},
+                {"x":15.25, "y":0},
+                {"x":16.25, "y":0},
+                {"x":17.25, "y":0},
+                {"x":18.25, "y":0},
+                {"x":19.75, "y":0},
+                
+                {"x":0, "y":1},
+                {"x":1, "y":1},
+                {"x":2, "y":1},
+                {"x":3, "y":1},
+                {"x":4.25, "y":1, "w":1.5},
+                {"x":5.75, "y":1},
+                {"x":6.75, "y":1},
+                {"x":7.75, "y":1},
+                {"x":8.75, "y":1},
+                {"x":9.75, "y":1},
+                {"x":10.75, "y":1},
+                {"x":11.75, "y":1},
+                {"x":12.75, "y":1},
+                {"x":13.75, "y":1},
+                {"x":14.75, "y":1},
+                {"x":15.75, "y":1},
+                {"x":16.75, "y":1},
+                {"x":17.75, "y":1, "w":1.5},
+                {"x":19.75, "y":1},
+                
+                {"x":0, "y":2},
+                {"x":1, "y":2},
+                {"x":2, "y":2},
+                {"x":3, "y":2},
+                {"x":4.25, "y":2, "w":1.75},
+                {"x":6, "y":2},
+                {"x":7, "y":2},
+                {"x":8, "y":2},
+                {"x":9, "y":2},
+                {"x":10, "y":2},
+                {"x":11, "y":2},
+                {"x":12, "y":2},
+                {"x":13, "y":2},
+                {"x":14, "y":2},
+                {"x":15, "y":2},
+                {"x":16, "y":2},
+                {"x":17, "y":2},
+                {"x":18, "y":2, "w":1.25},
+                {"x":19.75, "y":2},
+                
+                {"x":0, "y":3},
+                {"x":1, "y":3},
+                {"x":2, "y":3},
+                {"x":3, "y":3},
+                {"x":4.25, "y":3, "w":1.25},
+                {"x":5.5, "y":3},
+                {"x":6.5, "y":3},
+                {"x":7.5, "y":3},
+                {"x":8.5, "y":3},
+                {"x":9.5, "y":3},
+                {"x":10.5, "y":3},
+                {"x":11.5, "y":3},
+                {"x":12.5, "y":3},
+                {"x":13.5, "y":3},
+                {"x":14.5, "y":3},
+                {"x":15.5, "y":3},
+                {"x":16.5, "y":3, "w":1.75},
+                {"x":18.5, "y":3.25},
+                {"x":19.75, "y":3},
+                
+                {"x":0, "y":4},
+                {"x":1, "y":4},
+                {"x":2, "y":4},
+                {"x":3, "y":4},
+                {"x":4.25, "y":4, "w":1.25},
+                {"x":5.5, "y":4, "w":1.25},
+                {"x":6.75, "y":4, "w":1.25},
+                {"x":8, "y":4, "w":6.25},
+                {"x":14.25, "y":4, "w":1.5},
+                {"x":15.75, "y":4, "w":1.5},
+                {"x":17.5, "y":4.25},
+                {"x":18.5, "y":4.25},
+                {"x":19.5, "y":4.25}
+            ]
+        },
+        "LAYOUT_iso_numpad_left_split_bs": {
```

Comment:
The following layouts do not exist within `wyvern.h`, and should either be implemented or removed.

Reply:
For the record, I would be okay with merging this PR without this change, and having to personally come back around to add that later.